src/admin/resolvers/queries/IABCategory.integration.ts

```diff
@@ -86,14 +87,14 @@ describe('queries: IABCategory', () => {
       });
 
       // all the props of the first parent IAB category
-      expect(data[0].externalId).toBeTruthy();
-      expect(data[0].name).toBeTruthy();
-      expect(data[0].slug).toBeTruthy();
+      expect(data[0].externalId).not.to.be.null;
```

Comment:
Technically these assertions are different than they were before, since this is strict null equality. If you want to check for null or undefined, you can use `exist`

Reply:
i was looking for an equivalent to `toBeTruthy` - `exist` looks like it! ty!